docs/developer/decision-records/2022-06-28-entities-timestamp/README.md

```diff
@@ -0,0 +1,29 @@
+# Entities timestamp
+
+## Decision
+
+From the class `StatefulEntity`an abstract class called `Entity` will be extracted and become its superclass. Some
+entity classes created during the creation of policies and contracts will extend to this class `Entity`.
+
+The classes `TransferProcess` and `ContractNegotiation` will keep extended to the class `StatefulEntity` using all the
+inherited attributes from this one.
+
+## Rationale
```

Comment:
I would add a little explanation of what an entity is, also a quote from the [DDD page of wikipedia](https://en.wikipedia.org/wiki/Domain-driven_design#Kinds_of_models) would be enough:
> An entity is an object defined not by its attributes, but its identity.

Also a list of our current entities would be useful